src/vars.h

```diff
@@ -145,7 +166,7 @@ static inline void SetBrkCallTo( Expr expr, const char * file, int line ) {
             (int)expr, (int)STATE(CurrLVars), file, line);
   }
 #endif
-  STAT_LVARS_PTR(STATE(PtrLVars)) = expr;
+  ((LVarsHeader *)STATE(PtrLVars))->stat = expr;
```

Comment:
Would it make sense to have an eg `SET_STAT_LVARS_PTR` to use instead of this line?

Reply:
Ah, I thought it would also make it easier to grep for. But  one can also just grep for `STATE(PtrLVars))->stat`